src/white-bread.md

```diff
@@ -0,0 +1,38 @@
+# White Sandwich Bread
+
+![White Bread](pix/white-bread.webp)
+
+Simple white bread. Like the bread you get from the grocery store but 100x better.
+
+- ⏲️ Prep time:     10 min
+- ⬆️ Proofing time: 1.5 - 2.5 hrs
+- 🍞Baking time:   35 min - 1 hr
+
+## Ingredients
+
+- 3/4 cup (174g) water
+- 1/2 cup (125g) whole milk
+- 1 packet (9g) active dry yeast (or 6.75g - you can round up to 7g if your scale doesn't have decimals -  instant yeast)
```

Comment:
extra space after second `-`. also, what is the difference between 9g and 6.75g yeast?

Reply:
That is 9g of active dry yeast or 6.75g of instant yeast. Two different types of yeast